docs/source/guides/networking.rst

```diff
@@ -0,0 +1,144 @@
+Networking between VMs
```

Comment:
In my opinion this could use a better title.
It's not immediately obvious how the term "networking" applies to UI <-> Client VM data transfer.
It's fine if you want to keep it that way though, i'm struggling to come up with a better title.
That being said, I do think this should be moved into the Squirrel section

Reply:
Maybe "Communicating between VMs" would be a better title?

Reply:
"Communicating between CLIENT, UI and SERVER scripts"